gensim/models/doc2vec.py

```diff
@@ -580,7 +580,7 @@ def __init__(self, documents=None, dm_mean=None,
         need about 1GB of RAM. Set to `None` for no limit (default).
 
         `sample` = threshold for configuring which higher-frequency words are randomly downsampled;
-                default is 0 (off), useful value is 1e-5.
+                default is 1e-3, useful value is 1e-5.
```

Comment:
Thanks for the fix.

This doc reads very strange now: `default value is X, useful value is Y`. Why not use the useful value? 🤔
 @gojomo